include/grpc++/server_credentials.h

```diff
@@ -64,6 +64,7 @@ struct SslServerCredentialsOptions {
   };
   grpc::string pem_root_certs;
   std::vector<PemKeyCertPair> pem_key_cert_pairs;
+  bool force_client_auth;
```

Comment:
If we do not add a default ctor, then all the existing uses of the SslServerCredentialsOptions will need to be looked at, right?